chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChips.kt

```diff
@@ -0,0 +1,256 @@
+     */
+    @ExperimentalMaterialApi
+    @Composable
+    fun Input(
+        label: String,
+        onClick: (Boolean) -> Unit,
+        selected: Boolean,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        startContent: (@Composable () -> Unit)? = null,
+        endContent: (@Composable VitaminChipButtons.() -> Unit)? = null,
+        colors: ChipColors = VitaminChipColors.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand
+    ) {
+        VitaminChipImpl(
+            label = label,
+            onClick = { onClick(selected) },
+            selected = selected,
+            modifier = modifier,
+            enabled = enabled,
+            startContent = startContent,
+            endContent = endContent,
+            colors = colors,
+            sizes = sizes,
+            ripple = ripple
+        )
+    }
+
+    /**
+     * Chips appear dynamically as a group of multiple interactive elements.
+     * With chips, users can filter content, enter information, make selections or trigger actions.
+     *
+     * @param label Text to be displayed into the Chip
+     * @param onClick Callback to be called when the action Chip is clicked
+     * @param selected True if the Chip is selected, otherwise false
+     * @param `Modifier` to be applied to the component
+     * @param enabled True if you can click on the Chip, otherwise false
+     * @param startContent Optional Content to be displayed before label into the Chip
+     * @param endContent Optional Content to be displayed after label into the Chip
+     * @param colors Colors to be applied to the Chip
+     * @param sizes Sizes to be applied to the Chip. (VitaminChipSizes.medium() | VitaminChipSizes.small())
+     * @param ripple The ripple effect to be applied to the Chip
+     */
+    @ExperimentalMaterialApi
+    @Composable
+    private fun VitaminChipImpl(
+        label: String,
+        onClick: () -> Unit,
+        selected: Boolean,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        startContent: (@Composable () -> Unit)? = null,
+        endContent: (@Composable VitaminChipButtons.() -> Unit)? = null,
+        colors: ChipColors = VitaminChipColors.default(),
+        sizes: ChipSizes = VitaminChipSizes.medium(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand
+    ) {
+        val backgroundColor by colors.backgroundColor(selected = selected, enabled = enabled)
+        val borderColor by colors.borderColor(selected = selected, enabled = enabled)
+        val border = BorderStroke(sizes.borderWidth, borderColor)
+
+        val contentColor = colors.contentColor(selected = selected, enabled = enabled).value
+
+        CompositionLocalProvider(
+            LocalRippleTheme provides ripple,
+            LocalContentAlpha provides contentColor.alpha
+        ) {
+            Surface(
```

Comment:
You don't use the chip component from Compose? Not adapted to our cases?

Reply:
Changed using Chip & FilterChip Components 
Needed to override a bit for colors & spacings